packages/vitest/src/node/workspace.ts

```diff
@@ -297,7 +297,15 @@ export class WorkspaceProject {
   }
 
   async setServer(options: UserConfig, server: ViteDevServer) {
-    this.config = resolveConfig(this.ctx.mode, options, server.config)
+    this.config = resolveConfig(
```

Comment:
Maybe we should add a warning here if `options.coverage` is set?

Reply:
Would that come from workspace project's configuration? I don't think we have any runtime warnings for non-project options.

Reply:
Hm, yes, I remember. We don't have warnings because people can merge the main config with the project config.

Reply:
Silently ignoring the unsupported options seems like best option for that.